tests/integration/test_script_runner.py

```diff
@@ -156,6 +156,12 @@ class BackendDependencies(enum.Enum):
 """
 
 integration_test_matrix = [
+    {
+        "name": "rule_base_profiler_multi_batch_example",
+        "data_context_dir": "tests/integration/fixtures/yellow_trip_data_pandas_fixture/great_expectations",
+        "data_dir": "tests/test_sets/taxi_yellow_trip_data_samples",
+        "user_flow_script": "tests/integration/docusaurus/rule_based_profiler/multi_batch_example.py",
+    },
```

Comment:
Because this is used in docs, please move this dictionary to the `docs_test_matrix` for clarity.